themes/powerline/powerline.base.sh

```diff
@@ -192,7 +192,11 @@ function __powerline_prompt_command {
     local info="$(__powerline_${segment}_prompt)"
     [[ -n "${info}" ]] && __powerline_left_segment "${info}"
   done
-  [[ "${last_status}" -ne 0 ]] && __powerline_left_segment $(__powerline_last_status_prompt ${last_status})
+
+  ## info status prompt ##
+  info="$(__powerline_last_status_prompt ${last_status})"
```

Comment:
Thank you for updating. Please make `info` local. The variable `info` is not localized when there are no entries in the array `POWERLINE_PROMPT_ARRAY` because `local info` in the loop is not executed.
```suggestion
  local info="$(__powerline_last_status_prompt ${last_status})"
```

Or, another possibility is to declare `local info` before the for loop.

Reply:
Ready.